conf/machine/include/rpi-default-providers.inc

```diff
@@ -7,10 +7,10 @@ PREFERRED_PROVIDER_virtual/libgles2 ?= "${@bb.utils.contains("MACHINE_FEATURES",
 PREFERRED_PROVIDER_virtual/libgl ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
 PREFERRED_PROVIDER_virtual/mesa ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
 PREFERRED_PROVIDER_virtual/libgbm ?= "${@bb.utils.contains("MACHINE_FEATURES", "vc4graphics", "mesa", "mesa-gl", d)}"
-PREFERRED_PROVIDER_vlc ?= "rpidistro-vlc"
-PREFERRED_PROVIDER_ffmpeg ?= "rpidistro-ffmpeg"
-PREFERRED_PROVIDER_libav ?= "rpidistro-ffmpeg"
-PREFERRED_PROVIDER_libpostproc ?= "rpidistro-ffmpeg"
+PREFERRED_PROVIDER_vlc ?= "vlc"
```

Comment:
Shouldn't we drop these? These are default values, anyway.

Reply:
Good point fixed